src/api/PrincipalsApi.js

```diff
@@ -216,6 +217,27 @@ function getUsers(userIds :string[]) :Promise<Object> {
     });
 }
 
+/**
+* `POST /principals/db/credential`
+*
+* @static
+* @memberof lattice.PrincipalsApi
+* @returns {Promise} - a Promise that resolves without a value
+*
+* @example
+* PrincipalsApi.regenerateCredential();
+*/
+function regenerateCredential() :Promise<*> {
```

Comment:
How do you feel about naming this to `regenerateAtlasCredentials` since it's sibling is `getAtlasCredentials`?

Reply:
*we decided to leave it as is*